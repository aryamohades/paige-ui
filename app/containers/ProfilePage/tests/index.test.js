import React from 'react';
import renderer from 'react-test-renderer';

import { ProfilePage } from '../index';

describe('<AppHeader />', () => {
  it('should match the snapshot if user is not authenticated', () => {
    const renderedComponent = renderer.create(<ProfilePage />).toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });
});
