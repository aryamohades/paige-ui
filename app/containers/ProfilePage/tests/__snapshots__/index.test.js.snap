// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppHeader /> should match the snapshot if user is not authenticated 1`] = `
<div>
  Profile
</div>
`;
